crates/daphne/src/vdaf/mastic.rs

```diff
@@ -54,11 +56,11 @@ pub enum MasticWeight {
     Bool(bool),
 }
 
-pub(crate) fn mastic_shard(
+pub(crate) fn mastic_shard<const NUM_AGGREGATORS: usize>(
```

Comment:
Why do we need this? NUM_AGGREGATORS will always be 2 in DAP, so i think you could just hard-code 2 in the result below.

Reply:
maybe I was a bit too fancy :sweat_smile: